frontend/packages/schema-editor/src/components/SchemaInspector/ItemFieldsTab/ItemFieldsTable/ItemFieldsTableRow.tsx

```diff
@@ -78,8 +96,7 @@ export const ItemFieldsTableRow = ({
         />
       </td>
       <td className={cn(classes.tableColumnType, classes.tableCell)}>
-        {isField(fieldNode) &&
-          typeOptions.find(({ value }) => value === fieldNode.fieldType)?.label}
+        {typeLabel || kindLabel || (isReference(node) && <ReferenceButton node={node} />) || ''}
```

Comment:
Jeg liker godt at du har lagt til en lenke til typen når det er en referanse. Men jeg er ikke sikker på `ReferenceButton` er riktig komponent. Som du kan se, ligger den i `SchemaNode`-mappen, og den blir ikke eksponert gjennom `index.ts`. Det er fordi den er ment for å brukes intert i `SchemaNode`, som igjen kun brukes internt i `SchemaTree`. Vi bør ikke importere noe annet fra denne mappen enn komponenten som helhet. Det er viktig for å ikke ende opp med "spaghettikode" med avhengigheter på kryss og tvers.

For å kunne ta i bruk `ReferenceButton`-komponenten, bør vi derfor flytte den opp på rotnivå i `components`-mappen. Men jeg er usikker på om det er den beste løsnignen her. Som du sikkert kan se når du tester, har den en stil som skiller seg fra andre knapper, som er laget for at den ikke skal avlede for mye oppmerksomhet i trevisningen. Her tror jeg en vanlig knapp er bedre. Jeg har to forslag:
- La `ReferenceButton` være som den er og lag en ny komponent for knappen som skal vises her
- Lag en mer generisk komponent som er en knapp for referanser, og ta den i bruk både her og i `ReferenceButton`

Reply:
Jeg lagt generisk komponent `StudioReferenceButton`